From 6542ee6bc6ce420dcb2c1967c63132933d2d2f33 Mon Sep 17 00:00:00 2001
From: Javier Llorente <javier@opensuse.org>
Date: Tue, 8 Dec 2020 14:08:11 +0100
Subject: [PATCH] Fix application output

There is no application output on Qt Creator because logging goes to journald.
Workaround: set QT_LOGGING_TO_CONSOLE and QT_FORCE_STDERR_LOGGING
---
 src/app/main.cpp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/app/main.cpp b/src/app/main.cpp
index bc5e61c7..c6809869 100644
--- a/src/app/main.cpp
+++ b/src/app/main.cpp
@@ -549,6 +549,9 @@ int main(int argc, char **argv)
         QApplication::setAttribute(Qt::AA_DontUseNativeMenuBar);
     }
 
+    // Force Qt to log to stderr
+    qputenv("QT_FORCE_STDERR_LOGGING", "1");
+
     Utils::TemporaryDirectory::setMasterTemporaryDirectory(QDir::tempPath() + "/" + Core::Constants::IDE_CASED_ID + "-XXXXXX");
 
 #ifdef Q_OS_MACOS
-- 
2.31.1

